qiskit/qasm3/__init__.py

```diff
@@ -172,13 +172,65 @@
     \"\"\"
     circuit = qiskit.qasm3.loads(program)
     circuit.draw("mpl")
+
+
+Experimental import interface
+-----------------------------
+
+The import functions given above rely on the ANTLR-based reference parser from the OpenQASM project
+itself, which is more intended as a language reference than a performant parser.  You need to have
+the extension ``qiskit-qasm3-import`` installed to use it.
+
+Qiskit is developing a native parser, written in Rust, which is available as part of the core Qiskit
+package.  This parser is still in its early experimental stages, so is missing features and its
+interface is changing and expanding, but it is typically orders of magnitude more performant for the
+subset of OpenQASM 3 it currently supports, and its internals produce better error diagnostics on
+parsing failures.
+
+You can use the experimental interface immediately, with similar functions to the main interface
+above:
+
+.. autofunction:: load_experimental
+.. autofunction:: loads_experimental
+
+These two functions are both experimental, meaning they issue an :exc:`.ExperimentalWarning` on
+usage, and their interfaces may be subject to change within the Qiskit 1.0.  In particular, the
```

Comment:
```suggestion
usage, and their interfaces may be subject to change within the Qiskit 1.x release series.  In particular, the
```

Reply:
oh lol, I totally meant to write that - it's why it says "within _the_ Qiskit" haha.  Done in f37f279e0a with line wrapping.